FuelLabs/fuel-indexer
# enhancement: make verbose logging configurable

### Description
- PR adds `-v --verbose` flag to `fuel-indexer` and `fuel-indexer-api-server`
- Updates docs accordingly

### Testing steps
- Ensure `forc-index start` points to a latest `fuel-indexer` binary
- Run with verbose logging
  - [ ] Try running the block explorer with and without the `-v --verbose` flag

 
```bash
cargo run --bin fuel-indexer -- run \
  --fuel-node-host beta-3.fuel.network \
  --fuel-node-port 80 \
  --manifest examples/block-explorer/explorer-indexer/explorer_indexer.manifest.yaml \
  --verbose
```
   - [ ] And the plugin as well (with and without verbosity)

```
cargo run --bin forc-index -- start -v
```

### Changelog
- enhancement: make verbose logging configurable
